roles/pulibrary.postgresql/tasks/create_users.yml

```diff
@@ -3,7 +3,7 @@
   postgresql_user:
     name: '{{ application_dbuser_name }}'
     login_host: '{{ postgres_host | default(omit) }}'
-    login_port: '{{ postgres_port | default(omit) }}'
+    port: '{{ postgres_port | default(omit) }}'
```

Comment:
@kayiwa here too.  Not entirely sure why...

Reply:
same here. https://docs.ansible.com/ansible/latest/modules/postgresql_user_module.html